src/components/Search/index.tsx

```diff
@@ -159,9 +237,80 @@ const Search = ({ dispatch, hasBorder, content }: { dispatch: AppDispatch; hasBo
           }
         }}
       />
+
       {hasBorder && <SearchIconButton greenIcon />}
+
+      {searchSuggestions.length > 0 && searchValue ? (
+        <SuggestionsDropdown
+          suggestions={searchSuggestions}
+          onSuggestionSelect={onSuggestionSelect}
+          expandSuggestions={expandSuggestions}
+          setExpandSuggestions={setExpandSuggestions}
+        />
+      ) : null}
     </SearchPanel>
   )
 }
 
+interface SearchSuggestion {
+  value?: string
+  balance?: number
+  path?: string
+  type: string
```

Comment:
Setting `type` data type `enum` maybe more better.

Reply:
How come? `type` will always be a `string`

Reply:
`enum` can also be initialized with `string`. https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
From the documentation: "string enum values need to be individually initialized". In this case, I'm just trying to ensure that `SearchSuggestion` objects have the properties `value`, `balance`, `path`, and `type` so I'm not totally sure how I would implement `enum` here.

Reply:
Nevermind! I see what you mean now!